### ISSUE TITLE
Pack Opener Inconsistent Ratios / Missing Cards  - Legacy of Darkness

### ISSUE THREAD

**Issue by heyitsthatdude17** [AlanOC91/YGOPRODeck#280]
While testing the pack opener for a sealed style tournament with friends, I noticed that over a large set of packs I was not seeing certain cards appear. Asking my friends and they noticed the same thing. I therefore ran a test where I opened 576 packs and reviewed the results. I have linked the the csvs containing my collection with all the pulls in it, and a second file where I calculated the pull ratios and looked at overall pulls. I compiled this with some math done by hand to combine 1st ed and unlimited pulls into one value, and because of that my values are off by ~100 cards missing. However the margins discussed are so wide this should not be a factor, and all the cards discussed as issue cards were double-checked. Here are my findings:

The cards marked as short print according to https://db.ygoprodeck.com/set/?search=Legacy%20of%20Darkness have wildly different pull ratios. Some of them seem to have the pull ratios of commons (Cave Dragon, Inaba White Rabbit, Disappear, Otohime) while others were non-existant in the over 5000 cards I pulled. Smoke Grenade of the Thief only appeared twice, which seems low even for a Super Short Print (and Smoke Grenade is only listed as regular short print). Maharagi was pulled 0 times out of the 500+ packs.

Additionally, one more outlier exists in the form of Gradius' Option. It is a common, but I pulled 0 throughout the entire opening. This one must be a missed card, as there's no feasible way a common could never be pulled.

Having not seen the codebase myself, I do not know if this issue with SP/SSP is a widespread issue or just relegated to LOD. LOD is the only pack I have tested thus far to this degree.

Collection
https://files.catbox.moe/w9v1lr.csv
Computed Ratios
https://files.catbox.moe/oefwo9.csv

**Comment by AlanOC91** [comment]
I think the issue here is definitely SP/SSP not being accounted for in my code. I completely overlooked that rarity. Because of how it's coded, it can appear in some instances but it's rare.

I'll make some modifications to accommodate this!

**Comment by AlanOC91** [comment]
Ok, Short Print and Super Short Print are now accounted for. However, I'm not overly pleased with the extremely low drop rates of these rarities so I'll be adjusting as necessary going forward. 

I found the issue with `Gradius' Option`. To cut a long story short we do a database join with the TCGplayer Database that we pull for prices/store links. They had `Gradius' Option` incorrectly spelled as `Gradius's Option`. So it would never have been pulled. I've rolled out a fix for that too. 

Most of these issues are usually a result of the join with the TCGplayer Database that I tend to need to retroactively fix once they are pointed out to me. For example, they also have the cards like` Cave Dragon` marked as a Common which fits with your findings.

I'll go through these and fix them as best I can.
